okapi-core/src/main/java/org/folio/okapi/bean/TenantModuleDescriptor.java

```diff
@@ -33,6 +33,10 @@
     pending, deploy, invoke, undeploy, done
   }
 
+  // if true, it specifies that the id is specified by user and can not be changed
+  @JsonIgnore
+  private Boolean fixed;
```

Comment:
I don't see the need why this should have three values: true, false, null. Can a boolean (with true and false only) been used instead?

Reply:
True. I'll change that.